com.unity.multiplayer.mlapi/Editor/ProfilerModuleOverride/ProfilerInfo.cs

```diff
@@ -0,0 +1,126 @@
+                ProfilerMarkerDataUnit.Count, ProfilerCounterOptions.FlushOnEndOfFrame);
+
+        public static MLAPIProfilerCounterValue<int> BytesReceivedCounterValue =
+            new MLAPIProfilerCounterValue<int>(ProfilerCategory.Network, ProfilerConstants.NumberBytesReceived.ToString(),
+                ProfilerMarkerDataUnit.Count, ProfilerCounterOptions.FlushOnEndOfFrame);
+
+        public static MLAPIProfilerCounterValue<int> NetworkVarsCounterValue =
+            new MLAPIProfilerCounterValue<int>(ProfilerCategory.Network, ProfilerConstants.NumberNetworkVarsReceived.ToString(),
+                ProfilerMarkerDataUnit.Count, ProfilerCounterOptions.FlushOnEndOfFrame);
+
+        //RPCs
+        public static MLAPIProfilerCounterValue<int> RPCsSentCounterValue =
+            new MLAPIProfilerCounterValue<int>(ProfilerCategory.Network, ProfilerConstants.NumberOfRPCsSent.ToString(),
+                ProfilerMarkerDataUnit.Count, ProfilerCounterOptions.FlushOnEndOfFrame);
+
+        public static MLAPIProfilerCounterValue<int> RPCsReceivedCounterValue =
+            new MLAPIProfilerCounterValue<int>(ProfilerCategory.Network, ProfilerConstants.NumberOfRPCsReceived.ToString(),
+                ProfilerMarkerDataUnit.Count, ProfilerCounterOptions.FlushOnEndOfFrame);
+
+        public static MLAPIProfilerCounterValue<int> RPCBatchesSentCounterValue =
+            new MLAPIProfilerCounterValue<int>(ProfilerCategory.Network, ProfilerConstants.NumberOfRPCBatchesSent.ToString(),
+                ProfilerMarkerDataUnit.Count, ProfilerCounterOptions.FlushOnEndOfFrame);
+
+        public static MLAPIProfilerCounterValue<int> RPCBatchesReceivedCounterValue =
+            new MLAPIProfilerCounterValue<int>(ProfilerCategory.Network, ProfilerConstants.NumberOfRPCBatchesReceived.ToString(),
+                ProfilerMarkerDataUnit.Count, ProfilerCounterOptions.FlushOnEndOfFrame);
+
+        public static MLAPIProfilerCounterValue<int> RPCQueueProcessedCounterValue =
+            new MLAPIProfilerCounterValue<int>(ProfilerCategory.Network, ProfilerConstants.NumberOfRPCQueueProcessed.ToString(),
+                ProfilerMarkerDataUnit.Count, ProfilerCounterOptions.FlushOnEndOfFrame);
+
+        public static MLAPIProfilerCounterValue<int> RPCsInQueueSizeCounterValue =
+            new MLAPIProfilerCounterValue<int>(ProfilerCategory.Network, ProfilerConstants.NumberOfRPCsInQueueSize.ToString(),
+                ProfilerMarkerDataUnit.Count, ProfilerCounterOptions.FlushOnEndOfFrame);
+
+        public static MLAPIProfilerCounterValue<int> RPCsOutQueueSizeCounterValue =
+            new MLAPIProfilerCounterValue<int>(ProfilerCategory.Network, ProfilerConstants.NumberOfRPCsOutQueueSize.ToString(),
+                ProfilerMarkerDataUnit.Count, ProfilerCounterOptions.FlushOnEndOfFrame);
+
+
+
+        public static readonly Guid kNetworkingOperationsProfilerModuleGUID = new Guid("42c5aec2-fb86-4172-a384-34063f1bd332");
```

Comment:
nit, pretty sure `k` prefix is only for privates

Reply:
Removed